README.md

```diff
@@ -6,7 +6,7 @@ Solutions to [Project Euler Problems](https://projecteuler.net/archives)
 * Each solution should be confined in it's own solution folder.
 * Solution folders should pass the following regular expression:
 
-`problem[0-9]+-(bash|c|cpp|cs|fsharp|java|go|haskell|js|php|python|ruby|swift|groovy)$`.
+`problem[000-999]+-(bash|c|cpp|cs|fsharp|java|go|haskell|js|php|python|ruby|swift|groovy)$`.
```

Comment:
This regular expression should be adjusted: `problem[0-9]{3}` instead of `problem[000-999]` to validate the desired results.